tsl/src/continuous_aggs/create.c

```diff
@@ -1513,12 +1512,14 @@ finalizequery_get_select_query(FinalizeQueryInfo *inp, List *matcollist,
 		ColumnDef *cdef = (ColumnDef *) lfirst(lc);
 		Value *attrname = makeString(cdef->colname);
 		rte->eref->colnames = lappend(rte->eref->colnames, attrname);
+		rte->selectedCols =
+			bms_add_member(rte->selectedCols,
```

Comment:
this should be bms_add_member(rte->selectedCols, attno)?

Reply:
is it possible to use bms_add_members to add them all together?

Reply:
the numbers in the bitmapset are offset to allow for systemcolumns to be represented as well. bitmapsets can only store positive numbers but systemcolumns are negative so postgres uses an offset here to be able to represent them as well.